components/tableNodes/NodeTable.component.tsx

```diff
@@ -55,6 +55,19 @@ export function NodeTable({ show }: NodeTableProps): JSX.Element {
     show(true, "No channels open in this node");
     return <></>;
   }
+
+  function convert(param: any): number {
+    if (param instanceof Number) {
+      return param as number;
+    }
+    return parseInt(param);
+  }
+  function check_ammount_msat(param: any): string {
+    if (param instanceof Number) {
+      return "msats";
+    }
+    return "";
+  }
```

Comment:
This is a buggy function!

Maybe better this
```suggestion
  function check_ammount_msat(param: any): string {
  // if it is a number we return the value by appending msat
    if (param instanceof Number) {
      return `${param} msats`;
    } else {
    if (!params.contains("msat")) { 
        return `${param} msats`;
    }
     return param;
    }
  }
```

please check if there are typo

Reply:
we have `.includes` instead of `.contains`